pkg/logentry/stages/template.go

```diff
@@ -89,67 +87,37 @@ func (o *templateStage) Process(labels model.LabelSet, extracted map[string]inte
 	if o.cfgs == nil {
 		return
 	}
-	if v, ok := extracted[o.cfgs.Source]; ok {
+	td := make(map[string]interface{})
+	for k, v := range extracted {
 		s, err := getString(v)
 		if err != nil {
 			if Debug {
 				level.Debug(o.logger).Log("msg", "extracted template could not be converted to a string", "err", err, "type", reflect.TypeOf(v).String())
 			}
 			return
 		}
-		td := templateData{s}
-		buf := &bytes.Buffer{}
-		err = o.template.Execute(buf, td)
-		if err != nil {
-			if Debug {
-				level.Debug(o.logger).Log("msg", "failed to execute template on extracted value", "err", err, "value", v)
-			}
-			return
-		}
-		st := buf.String()
-		// If the template evaluates to an empty string, remove the key from the map
-		if st == "" {
-			delete(extracted, o.cfgs.Source)
-		} else {
-			extracted[o.cfgs.Source] = st
+		td[k] = s
+		if k == o.cfgs.Source {
+			td["Value"] = s
 		}
```

Comment:
@cyriltovena I was missing this point of backward compatibility for `Value`. Thank you so much